### PR TITLE
[RFC] Restructure GNOME metapackage

### PR DESCRIPTION
**Pull request by not-chicken**
This PR depends on #21382

Discussion continued from https://github.com/void-linux/void-packages/pull/21382#issuecomment-627240142

### CODE REVIEW

**Comment by flexibeast** [comment]
So should https://github.com/void-linux/void-docs/pull/271 be blocked on this, or only on #21382?

**Comment by not-chicken** [comment]
> So should [void-linux/void-docs#271](https://github.com/void-linux/void-docs/pull/271) be blocked on this, or only on #21382?

Yes this one too.

File: srcpkgs/gnome-core/template [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ network-manager-applet>=1.8.24_1
+ orca>=3.36.2_1
+ pango>=1.42.4_7
+ pangomm>=2.42.0_1
+ phodav>=2.3_2
+ python-atspi>=2.36.0_2
+ python-gobject>=3.36.0_1
+ rest>=0.8.1_3
+ tracker>=2.3.4_1
+ vala>=0.46.3_1
```
Comment (sgn) [inline srcpkgs/gnome-core/template:110]: Doesn it need vala, and yeld-tools?
I think it only requires libvala,

Comment (not-chicken) [reply srcpkgs/gnome-core/template:110]: Sorry for the late reply. 

Deciding what to add in `gnome-core` and what not is pretty subjective and hence open to discussion. As a reference GNOME considers these as "core" https://ftp.gnome.org/pub/GNOME/core/3.36/3.36.2/sources/. packages, which I've tried to replicate with `gnome` meta-package.

Personally, I am also in favor of moving these two to `gnome`.

**Comment by not-chicken** [comment]
Would it be ok to pull `mesa-dri` through either of these meta-packages? It's required in order to have a working graphical session via wayland.

**Comment by ProjectMoon** [comment]
gnome-screensaver should be added to the metapackage. Without it, screen locking does not work.

**Comment by not-chicken** [comment]
`gnome-screensaver` or any other screen locker is needed only if GNOME is not started with GDM.

If you're indeed starting GNOME with GDM, then try disabling shell extensions if you have any.

-------- Original Message --------

**Comment by ProjectMoon** [comment]
Hmm well this was with budgie desktop, which uses GNOME screensaver

**Pull request closed without merging**